allinbits/cosmos-cash
# Feat/burn token

### Description

An issuer can burn emoney tokens

closes: #87

### How to test

- `make start-dev`
- `make seed`
- `cosmos-cashd query bank total --output json | jq` => check the supply
- `cosmos-cashd tx issuer burn-token 9997 --from validator --chain-id cash -y`
